config/lobby_notices.json

```diff
@@ -0,0 +1,3 @@
+[
+  "This is an example of speex, an audio compression codec specifically tuned for the reproduction of human speech."
```

Comment:
whar

Reply:
this was my testing config file, it doesn't entirely matter because the server doesn't download new config files.

Reply:
changed it so its empty.